cypress/tests/data/60-content/AmwandengaSubmission.cy.js

```diff
@@ -560,12 +559,15 @@ describe('Data suite: Amwandenga', function() {
 		cy.clickStageParticipantButton('Stephanie Berardo', 'Login As');
 		cy.get('button').contains('OK').click();
 		cy.openWorkflowMenu('Title & Abstract')
+		cy.wait(1000); // let the UI load fully
+	  
 		cy.get('button:contains("Publish")').should('not.exist');
 		cy.get('button:contains("Create Version")').should('not.exist');
-		cy.get('button').contains('All Versions').click();
-		cy.get('button').contains('Version 1').click();
-		cy.contains('This version has been published and can not be edited.');
```

Comment:
I'm not sure if the new warning text appears here -- but if it does, there should be a Cypress check for it.